### ISSUE TITLE
VSCode exit hangs W10 

### ISSUE THREAD

**Issue by rasq** [microsoft/vscode#81548]
- VSCode Version: all 
- OS Version: W10x64 1903

Steps to Reproduce:
1. Open Visual Studio code
2. Close Visual Studio code

Additional informations:
- installed Slack, Webex
Before 1903 there was not any similar issues. 
System seems to be working, after VSCode exit iam unable to run new webbrowser page, ping any ip adress etc. or open new application. System itself it working.
Problem entountered on many devices in my company. 
Devices in domain, InTune in use. Main antywirus ms defender.
Hardware: Lenovo laptops. 

Does this issue occur when all extensions are disabled?: Yes
Does this issue occur when GPU disabled from shortcut?: Yes
Does this issue occur when telemetry disabled?: Yes
Does this issue occur when git disabled?: Yes

**Comment by thapps** [comment]
Issue description

	While using Visual Studio Code, all application windows tends to randomly freeze with the permanent "(not responding)" status in the titlebar, rendering whole system to be unusable and requiring a restart

System setup

	VSCode Version: 1.38.1 (system setup) / (user setup), and all previous versions from ~half year or more
	OS Version: W10x64 1903, W10x64 1809

Steps to Reproduce:

	Not determined / happens randomly
    
	Favorable events to reproduce it:
	- opening and closing Visual Studio code
	- opening files for editing in Visual Studio code using r-click context menu
	- (rarely) switching focus/minimizing/restoring Visual Studio code windows

Additional information:

	Other Electron-based apps, like Slack are working fine when VSCode is not used
	Issue started around 1809 feature update relase, before that there was no similar issues.
	System seems to be working but there is no way to recover from this state other than restarting
	Laptop models: Lenovo T460 / T480    
	Display adapter: Intel(R) HD Graphics 520

	Does this issue occur when all extensions are disabled?: Yes
	Does this issue occur when GPU disabled from shortcut?: Yes
	Does this issue occur when telemetry disabled?: Yes
	Does this issue occur when git disabled?: Yes

**Comment by chrmarti** [comment]
/needsMoreInfo

Does this also happen with VS Code Insiders? Download: https://code.visualstudio.com/insiders/

We recently updated Insiders to Electron 6, please confirm you still see it with the latest.

**Comment by rasq** [comment]
Yes, this same issue with insiders.
I will try to małe video.
Are then aby logfiles for VSCode i can autach?

**Comment by bpasero** [comment]
We are building exploration builds that use a much newer version of our UI framework (Electron version 6.0.x). I wonder if this issue reproduces with one of these builds, could you try? Download:
* macOS: [Download](https://az764295.vo.msecnd.net/exploration/960208fdf9aa06e6892aa14c9c84ddd607fbeb2f/VSCode-darwin-exploration.zip)
* Linux (DEB): [Download](https://az764295.vo.msecnd.net/exploration/960208fdf9aa06e6892aa14c9c84ddd607fbeb2f/code-exploration_1.37.0-1564556412_amd64.deb)
* Linux (RPM): [Download](https://az764295.vo.msecnd.net/exploration/960208fdf9aa06e6892aa14c9c84ddd607fbeb2f/code-exploration-1.37.0-1564556601.el7.x86_64.rpm)
* Windows: [Download](https://az764295.vo.msecnd.net/exploration/960208fdf9aa06e6892aa14c9c84ddd607fbeb2f/VSCodeUserSetup-x64-1.37.0-exploration.exe)

**Comment by rasq** [comment]
@bpasero still the same issue.
Maybe it will be better to have build with some advanced loging? So i can upload logs and you can pinpoint whats messing around?

**Comment by bpasero** [comment]
Please see if you can find any logged output that could help us find the source of the issue:
* run `code --verbose` from the command line and attach the output
* in Code select **Help | Toggle Developer Tools** and select the **Console**
* attach any output you see in the **Console** you can find

Thanks!

**Comment by rasq** [comment]
I've done tests using stable and exploration version.
Both of version works simmilar. But i've noticed one think: if i dont run chrome, slack or other chrominium (electron) based software i can run VSCode once, secound one will crash like before. 

[logs.zip](https://github.com/microsoft/vscode/files/3713509/logs.zip)

**Comment by bpasero** [comment]
@rasq so only VSCode crashes or also Slack and other Electron apps?

**Comment by rasq** [comment]
@bpasero when i close VSCode generally whole W10 is crashed (all applications that use internet/network on't respond, ewen i cannot run ping from console). But i can close, open chrome, slack or other apps again without issues to moment when i close VSCode.

**Comment by bpasero** [comment]
Wow that sounds horrible, only a restart fixes that?

**Comment by rasq** [comment]
Yes, only restart. Maybe some complicated scipt that will restart some proceses will help, but i cannot know which one on what configuration.

**Comment by bpasero** [comment]
@deepak1556 every heard of this issue where closing VSCode when slack or chrome is running causes all network requests to stop working?

**Comment by csdivad** [comment]
I have the same issue.

Closing VSCode brakes Windows and some applications start to hang. Browsers do not open or not open properly, and eventually the whole system becomes unusable. The best thing is to restart the machine as soon as possible because even that may become difficult after time.

I don't think it is VSCode related but closing VSCode is the only way I was able to reproduce the problem. I have seen the same issue happening on my machine without VSCode involvment. VSCode actually disappears from the Task Manager upon exit.

I think it should be some kernel/driver problem, but I'm not able to investigate it properly (work machine). The machine is a ThinkPad T480 btw.

(Update: Is it possible that this got fixed in latest versions? I just opened VSCode and used the build in updater. After the update VSCode closed itself and Windows got broken. Restarted the OS and now it seems that I can't reproduce it anymore with new version. Or maybe I'm just lucky now?)
